xscen/data/fr/LC_MESSAGES/xscen.po

```diff
@@ -0,0 +1,35 @@
+# French translations for xscen.
+# Copyright (C) 2023 ORGANIZATION
+# This file is distributed under the same license as the xscen project.
+# Équipe Xscen <Rondeau-Genesse.Gabriel@ouranos.ca>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: xscen 0.6.18b0\n"
```

Comment:
Does it not update correctly, or is it only that you have not run it since more recent versions?

Reply:
This needs to be configured in the `setup.cfg for `bumpversion`. I can add that in here.